import-service/output.tf

```diff
@@ -0,0 +1,8 @@
+output "ATTENTION" {
+  value = <<EOF
+THIS PROFILE REQUIRES MANUAL STEPS!
+To run the manual steps run these scripts in the import-service directory:
+./setup_gae_firewall.py ${module.import-service-project.project_name} ${var.env}
+./add-import-service-sa-to-sam.sh ${var.env} ${data.google_service_account_access_token.sam_access_token.access_token} ${local.import_service_sa_email} ${data.google_service_account_access_token.import_service_token.access_token}
+EOF
+}
```

Comment:
I feel a bit wonky about outputting access tokens here. Thinking out loud ... is  there any risk to this? Whoever is running these scripts has a lot of privileged access anyway. Are there cases in which this output could be logged?

Reply:
the biggest risk is that it gets output into the github PR. however, this repo is private, and the tokens expire after 300s.